farbfeld.c

```diff
@@ -62,23 +64,26 @@ static void query(void)
                            load_args,
                            load_return_vals);
 
-    gimp_install_procedure("file_farbfeld_save",
+    gimp_register_file_handler_mime(LOAD_PROC, "image/farbfeld");
```

Comment:
The `"image/farbfeld"` string should probably be in a const or #define as well.

Reply:
Optimization for optimization? The bug is fixed, the rest is not important to me.

Not resolved.